telos-backend/demo.js

```diff
@@ -1,15 +1,39 @@
 const TodoTask = require('./src/models/todo_task');
 
+// Check if there are any TodoTask documents
 TodoTask.find().countDocuments((err, count) => {
+  // If there are none
```

Comment:
Thanks for the comments, though I would argue that these comments are detailing what the code says and probably unnecessary.

Reply:
Will remove